.github/workflows/ci.yaml

```diff
@@ -14,42 +14,46 @@ jobs:
   build:
     env:
       COVER: true
-      GOPATH: ${{ github.workspace }}
     runs-on: ubuntu-18.04
     steps:
 
-    - name: Check out code into the Go module directory
+    - name: Check out code
       uses: actions/checkout@v2
-      with:
-        path: ./src/github.com/${{ github.repository }}
```

Comment:
Just for my own clarification, we are removing this because we don't need to be in any particular directory when using go modules right?

Reply:
Yeah, thanks to the go module, it no longer depends on the location of the directory to build.